product_gtin/views/product_view.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" ?>
+<openerp>
+    <data>
+        <record id="product_template_search_view" model="ir.ui.view">
+            <field name="model">product.template</field>
+            <field name="inherit_id" ref="product.product_template_search_view" />
+            <field name="arch" type="xml">
+                <field name="name" position="attributes">
+                    <attribute name="filter_domain">['|', '|',
+                        ('default_code','ilike',self),('name','ilike',self),
+                        ('barcode','ilike',self)]
```

Comment:
Isn't barcode already searchable in standard?

Reply:
Anyways, this is breaking any other inheritance.

Reply:
It's not really necessary.